### PR TITLE
made keanu-examples tests shorter

### PR DESCRIPTION
**Pull request by migwellian**
Reduced the number of samples we take in the `keanu-examples` tests

### CODE REVIEW

File: keanu-examples/bayesianMethodsForHackers/src/main/java/com/examples/TextMessaging.java [hunk 1]
```diff
@@ -50,10 +50,10 @@ public static TextMessagingResults run() {
         BayesianNetwork net = new BayesianNetwork(textsForDay.getConnectedGraph());
         net.probeForNonZeroProbability(1000);
 
-        final int numSamples = 50000;
+        final int numSamples = 800;
```
Comment (christophernorth) [inline keanu-examples/bayesianMethodsForHackers/src/main/java/com/examples/TextMessaging.java]: By the looks of things - this is failing on Travis: Perhaps we need to make sure this test is deterministic? Or increase the number of samples slightly?

**Comment by wesleyw72** [comment]
Interesting that Travis is failing here, I have rerun on my machine ~10 times with no problem

File: keanu-examples/bayesianMethodsForHackers/src/main/java/com/examples/TextMessaging.java (hunk 1 above)
Comment (migwellian) [reply keanu-examples/bayesianMethodsForHackers/src/main/java/com/examples/TextMessaging.java]: It already has a `KeanuRandom.setDefaultRandomSeed(1)`

Comment (migwellian) [reply keanu-examples/bayesianMethodsForHackers/src/main/java/com/examples/TextMessaging.java]: `KeanuRandom` uses `org.apache.commons.math3.random.MersenneTwister` under the hood. Would you expect these to generate different numbers on different platforms (given the same seed)?

Comment (migwellian) [reply keanu-examples/bayesianMethodsForHackers/src/main/java/com/examples/TextMessaging.java]: I confirmed that our random seeds work the same on all platforms with a new test `KeanuRandomTest#randomNumberGenerationIsPlatformIndependent`.

I added `VertexId.ID_GENERATOR.set(1);` to every test and now it passes.

**Pull request merged**